llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -198,13 +216,18 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
 
   switch (Opc) {
   case TargetOpcode::G_ANYEXT:
+  case TargetOpcode::G_PTRTOINT:
   case TargetOpcode::G_TRUNC:
     MI.setDesc(TII.get(TargetOpcode::COPY));
-    return true;
+    return selectCopy(MI, MRI);
```

Comment:
MI is modified on line 221. If selectCopy returns false shouldn't we still return true?

Reply:
You're right. I think selectCopy should always return true here, so I'll just assert that.